### PR TITLE
feature/Registered usernames that are duplicates or invalid should trigger aggressive warning modal

### PR DESCRIPTION
**Pull request by Kacper-RF**
### Pull Request Checklist

- [x] I have linked this PR to related GitHub issue.
- [x] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### CODE REVIEW

**Review by leblowl — APPROVED** [review]
Looks good. Perhaps squashing changes before merging could be a good idea to keep the git commit history easy to read on develop.

**Comment by leblowl** [comment]
Might be good to get a quick review from @siepra also

File: packages/desktop/src/renderer/components/widgets/aggressiveWarningModal/AggressiveWarningModal.component.tsx
Comment (siepra) [inline packages/desktop/src/renderer/components/widgets/aggressiveWarningModal/AggressiveWarningModal.component.tsx:63]: Why didn't you reuse <AgressiveWarning> from 'AggresiveWarning.component'?

File: packages/desktop/src/renderer/components/widgets/aggressiveWarningModal/AggressiveWarningModal.component.tsx
Comment (siepra) [inline packages/desktop/src/renderer/components/widgets/aggressiveWarningModal/AggressiveWarningModal.component.tsx:48]: What's the difference between AggresiveWarningModal, AggresiveWarningContainer and AggresiveWarningComponent? I think so far we've been using a little bit different template for modals

File: packages/mobile/src/components/AggressiveWarning/AggressiveWarning.stories.tsx
Comment (siepra) [inline packages/mobile/src/components/AggressiveWarning/AggressiveWarning.stories.tsx:5]: You have to add this story to `.storybook/index.js` otherwise it won't be reachable

Comment (siepra) [reply packages/mobile/src/components/AggressiveWarning/AggressiveWarning.stories.tsx:5]: If you will, you can automate it to be included automatically like in desktop package I believe

**Review by siepra — APPROVED** [review]
Ideally I'd want desktop and mobile changes to be done in separate pull requests

**Comment by siepra** [comment]
It just came to me, the name "AgressiveWarning" is a little bit odd. It doesn't really tell what's going on. I think something like "DuplicatedUsernameWarning" would be more accurate

File: packages/mobile/src/components/AggressiveWarning/AggressiveWarning.component.tsx
Comment (leblowl) [inline packages/mobile/src/components/AggressiveWarning/AggressiveWarning.component.tsx:58]: @holmesworcester should we be more explicit here, e.g. "has registered a duplicate username"?

**Comment by Kacper-RF** [comment]
We already have something similar as `DuplicatedUsernameWarning` so nomenclature was changed to - `PossibleImpersonationAttack` (just like the title on the designs in Figma).

**Review by siepra — APPROVED** [review]

**Pull request merged**
